.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - 'master'
+  pull_request:
+    branches:
+      - 'master'
+
+jobs:
+  test:
+    name: Test Suite
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - python: python3.6
+            dist: ubuntu:bionic
+          - python: python3.8
+            dist: ubuntu:focal
+          #- python: python3.10
+          #  dist: ubuntu:jammy
```

Comment:
The error you had mentioned looks like this: https://github.com/mosquito/cysystemd/issues/36

A comment there recommends doing `pip install cysystemd` instead, and indeed the PyPI page for `systemd` actually links to the `cysystemd` repository. So maybe changing that in `run.sh` would make things work?

Reply:
Maybe? I'd say that is a separate change. My goal here is just to port the existing CI, not to improve/update the things we test. :)  I had to change to newer Ubuntu releases since the old ones are no longer available or stopped working with recent pip packages, but I don't plan to do any more here than is necessary to maintain the old level of test coverage.